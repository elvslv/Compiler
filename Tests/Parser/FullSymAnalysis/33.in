var i, j: integer;
r: real;
begin
	while r do
		j := 5;
end.