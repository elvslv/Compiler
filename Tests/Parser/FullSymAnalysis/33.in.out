var I : INTEGER;
var J : INTEGER;
var R : REAL;

13 4 Condition must be integer